get-dependency-list.js

```diff
@@ -31,6 +31,11 @@ module.exports = function(filename, serverless) {
         });
         filesToProcess.push(abs);
       } else {
+        const numBaseTokens = name.indexOf('@') === 0 ? 2 : 1;
```

Comment:
Looks like this logic is covered in a module:

https://github.com/mattdesl/require-package-name

I would suggest:

`const moduleName = requirePackageName(name.replace(/\\/, '/')))` (insulate against windows path separators)